composer.json

```diff
@@ -35,6 +35,7 @@
         "drupal/clamav":"1.1.0",
         "drupal/coi": "1.0.0",
         "drupal/config_override_core_fields": "^1.0",
+        "drupal/config_readonly":"^1.0",
```

Comment:
To ensure we're explicitly not going to inherit unexpected changes should we pin to `1.0-beta3` - at least until `1.0` is released. At that point we could move back to `^1.0`, but before that we will inherit any new releases and with it database updates etc.

Happy to get feedback on this, but I generally feel this should be pinned to the specified version in the PR title.